frontend/micro-ui/web/micro-ui-internals/packages/modules/dristi/src/pages/employee/admission/CaseFileAdmission.js

```diff
@@ -522,6 +523,52 @@ function CaseFileAdmission({ t, path }) {
     });
   };
 
+  const handleDelayCondonation = async (caseDetails) => {
+    const documents = caseDetails?.caseDetails?.delayApplications?.formdata?.[0]?.data?.condonationFileUpload?.document
+      ? caseDetails.caseDetails.delayApplications.formdata[0].data.condonationFileUpload.document.map((document) => {
+          return {
+            documentType: document?.documentType,
+            fileStore: document?.fileStore,
+            additionalDetails: {
+              name: document?.fileName,
+            },
+          };
+        })
+      : [];
+    const applicationReqBody = {
+      tenantId,
+      application: {
+        tenantId,
+        filingNumber,
+        cnrNumber: caseDetails?.cnrNumber,
+        cmpNumber: caseDetails?.cmpNumber,
+        caseId: caseDetails?.id,
+        // referenceId: isExtension ? null : orderDetails?.id || null,
+        createdDate: new Date().getTime(),
+        applicationType: "DELAY_CONDONATION",
```

Comment:
Add this application type in MDMS of application Type.
Also make sure you hide it when user goes to create submission page using "Make Submission" button

Reply:
done